packages/paste-core/components/table/stories/index.stories.tsx

```diff
@@ -944,6 +944,60 @@ RowHeaders.story = {
   name: 'Row headers',
 };
 
+export const StickyHeaders = (): React.ReactNode => {
+  return (
+    <Table>
+      <THead stickyHeader={true}>
+        <Tr>
+          <Th>Column 1</Th>
+          <Th>Column 2</Th>
+          <Th>Column 3</Th>
+        </Tr>
+      </THead>
+      <TBody>
+        {[...new Array(100).keys()].map((index) => (
+          <Tr key={index}>
+            <Td>Content</Td>
+            <Td>Content</Td>
+            <Td>Content</Td>
+          </Tr>
+        ))}
+      </TBody>
+    </Table>
+  );
+};
+
+StickyHeaders.story = {
+  name: 'Sticky headers',
+};
+
+export const BorderlessStickyHeaders = (): React.ReactNode => {
+  return (
+    <Table variant="borderless">
+      <THead stickyHeader={true}>
+        <Tr>
+          <Th>Column 1</Th>
+          <Th>Column 2</Th>
+          <Th>Column 3</Th>
+        </Tr>
+      </THead>
+      <TBody>
+        {[...new Array(100).keys()].map((index) => (
+          <Tr key={index}>
+            <Td>Content</Td>
+            <Td>Content</Td>
+            <Td>Content</Td>
+          </Tr>
+        ))}
+      </TBody>
+    </Table>
+  );
+};
+
+BorderlessStickyHeaders.story = {
+  name: 'Borderless Sticky headers',
+};
+
```

Comment:
suggestion: could we add a story for sticky headers that also uses `top`?

Reply:
Can and probably should

Reply:
Actually we can't. Because you can't actually see the top property apply until you scroll the page, which chromatic doesn't support

Reply:
Is this something cypress can do?

Reply:
Yes, but not in this situation. 

Cypress targets the website, and talks to Applitools. 

We have Stories and talk to Chromatic.